common/static/js/vendor/text.js

```diff
@@ -0,0 +1,390 @@
+/**
```

Comment:
I wonder if you should create a ``requirejs`` folder and move this and other libraries into it. As it stands, it is rather strange to have a vendor library just called 'text.js'.